### PR TITLE
Enhancement: Clean up configuration for friendsofphp/php-cs-fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] slightly cleans up the configuration for `friendsofphp/php-cs-fixer`

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -8,200 +8,198 @@ For the full copyright and license information, please view the LICENSE
...
+                'method',
+                'property',
             ],
-            'void_return' => true,
-            'whitespace_after_comma_in_array' => true,
-        ]
-    )
-    ->setFinder(
-        PhpCsFixer\Finder::create()
-        ->files()
```
Comment (localheinz) [inline .php_cs.dist]: Introducing an explaining variable allows us to get rid of the a bit awkward indentation here.

File: .php_cs.dist [hunk 2]
```diff
@@ -8,200 +8,198 @@ For the full copyright and license information, please view the LICENSE
...
-                    'require',
-                    'require_once',
-                    'return',
-                    'switch',
-                    'throw',
-                    'try',
-                    'while',
-                    'yield',
-                ],
+    ->setRules([
```
Comment (localheinz) [inline .php_cs.dist]: I have removed the wrapping of the argument list (which contains a single argument only) of `setRules()`, and instead wrap the argument itself only.

**Comment by sebastianbergmann** [comment]
Can you make this against `8.5`? I'll merge it from there to `master` then. Thanks!

**Comment by sebastianbergmann** [comment]
Can you make this against `8.5`? I'll merge it from there to `master` then. Thanks!

**Comment by localheinz** [comment]
@sebastianbergmann 

Done!

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann!

**Pull request merged**